src/modules/aix/server/dispatch/chatGenerate/adapters/openai.chatCompletions.ts

```diff
@@ -74,7 +77,7 @@ export function aixToOpenAIChatCompletions(openAIDialect: OpenAIDialects, model:
     tools: chatGenerate.tools && _toOpenAITools(chatGenerate.tools),
     tool_choice: chatGenerate.toolsPolicy && _toOpenAIToolChoice(openAIDialect, chatGenerate.toolsPolicy),
     parallel_tool_calls: undefined,
-    max_tokens: model.maxTokens !== undefined ? model.maxTokens : undefined,
+    max_completion_tokens: model.maxTokens !== undefined ? model.maxTokens : undefined,
```

Comment:
Look at the source for `TRequest`, that is where it says depreciated

Reply:
also as per docs 
![image](https://github.com/user-attachments/assets/415f8e4f-5006-4f9d-850d-385bc2d59396)